frontend/ui/app/[locale]/home/(about)/AboutSection.module.css

```diff
@@ -0,0 +1,33 @@
+.aboutContainer {
+    display: flex;
+    flex-direction: row;
```

Comment:
Flex is `row` by default, don't need to specify it explicitly

Reply:
Does this apply for react as well? switches to column if no explicit specification of row https://reactnative.dev/docs/flexbox

Reply:
It is just css. Does not matter if you write in a css module or inline. Flex is by default `row`